src/app/component/block/paragraph/data/data.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable */
+/* eslint-disable no-template-curly-in-string */
```

Comment:
Even though it's annoying to disable this, in most case we use `yaml` instead of `js` for data, and variables are not used that often. So I'm okay with this :)